index.html

```diff
@@ -15,15 +15,148 @@
 
     <!-- We are using a CSS Reset file to standardize browser default css -->
     <link rel="stylesheet" href="reset.css" />
-
-    <!-- This is where all of our awesome CSS code goes -->
     <link rel="stylesheet" href="style.css" />
+    <link href="Mina/Mina-Regular.ttf" rel="stylesheet"> <!-- this is the logo-->
   </head>
-  <body>
-    <!--
-
-      YOU HAVE TO MODIFY THE HTML TO BUILD YOUR AWESOME PAGE
+  <main>
+  <header>
+        <h1>MeLisa</h1>  <!--using the font "mina" from google for the logo-->
+        <h2>Lisa Jonas</h2> 
+        <h3 class="header-subtitle">PHOTOGRAPHER & DESIGNER</h3>
+        <nav>
 
-    -->
-  </body>
+          <span>Home</span>
+          <img src="dot.png" alt="dot">
+          <span>Store</span>
+          <img src="dot.png" alt="dot">
+          <span>Blog</span>
+          <img src="dot.png" alt="dot">
+          <span>Resume</span>
+          <img src="dot.png" alt="dot">
+          <span>Contact</span>
+          
+        </nav>
+    </header>
+<body>
+    <section>
+    <div class="paragraphs-container">
+        <div class="paragraph-square">
+          <h3 class="body-titles">Bio</h3>
+          <p>I'm a Photographer and designer 
+          from Montreal, living in LA.
+        When I'm not working, I'm probably drawing,
+        binge watching the latest hot series,
+        or taking photos of my cats</p></div>
+        <div class="paragraph-square">
+          <h3 class="body-titles">Expertise</h3>
+          <p>My work tends to be end-to-end product design and photography.
+         I've supported teams as a senior IC for years,
+         leading strategy and vision for complex systems, 
+         as well as strengthening culture and team growth.
+          </p></div>
+        <div class="paragraph-square">
+          <h3 class="body-titles">Work with me</h3>
+          <p>Shoot me a message below if you'd like to talk photography or design.
+             Here's my résumé.</p>
+        </div> 
+    </div>
+    
+    <div class="wavy-line">
+      <img src="wavy_line.png" alt="wavy line"><!--GO BACK AND CENTER THE LINE :(-->
+    </div>
+    </section>
+    <section>
+    <div class="Portfolio-container">
+      <h3 class="T-Portfolio">Portfolio</h3>
+      <p class="p-intro">Welcome to Lisa Jonas's Portfolio site. I am a newly graduated Photographer and Designer. 
+        I've found passion in capturing all of the smallest details that make our world a better place.</p>
+    </div>
+    <div class="portfolio-photos">
+      <img src="1.jpeg" alt="Photo 1">
+      <img src="2.jpeg" alt="Photo 2">
+      <img src="3.jpeg" alt="Photo 3">
+      <img src="4.jpeg" alt="Photo 4">
+    </div>
+    <div class="expand">
```

Comment:
You can improve this code by using an unordered list and then `list-style-type: none` to "list" all of the images

```html
<ul class="portfolio-photos">
  <li><img ...></li>
  <li><img ...></li>
  <li><img ...></li>
  <li><img ...></li>
</ul>